src/test/java/entities/ComputerPlayerTest.java

```diff
@@ -0,0 +1,76 @@
+package entities;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ComputerPlayerTest {
+
+    ComputerPlayer cp1;
```

Comment:
ditto for CardTest; add access modifiers

Reply:
Added in latest push.